Abstract Global status bits

LeandroMartinsdS opened the pull request:
Abstract the global status bits using the hardware interface.

Add PMAC_HARDWARE_PROB for PowerPMAC.
The `AbortAll` is not being used to define a "Hardware problem" because the PowerPMAC permits to abort just some of the coordinate systems, this is useful in cases which more than one system is controlled by the same controller, and just one should be affected by the Abort input - e.g. an air-bearing stage.

LeandroMartinsdS committed:
Abstract Global status bits

Add PMAC_HARDWARE_PROB for PowerPMAC

JamesOHeaDLS commented on pmacApp/src/pmacHardwarePower.cpp:
```diff
@@ -40,6 +40,36 @@ const int pmacHardwarePower::PMAC_STATUS1_IN_POSITION = (0x1 << 11);
+                                                   PMAC_GSTATUS_HW_CHANGE_ERROR |
+                                                   PMAC_GSTATUS_FILE_CONFIG_ERROR |
+                                                   PMAC_GSTATUS_NO_CLOCKS |
+                                                   //PMAC_GSTATUS_ABORTALL |
```
As well as commenting out PMAC_GSTATUS_ABORTALL, I think it would be worth including a comment in this section explaining why

Also, inline comments against each status bit explaining what it is would be useful (similar to what you did for the #defines in pmacController.h

JamesOHeaDLS reviewed (commented):
Looks good.  I only have the 1 suggestion about comments

LeandroMartinsdS committed:
Add comment to global statuses

Add description for each global status bits, together with the PowerPMAC
error
Add comment to explain why PMAC_GSTATUS_ABORTALL is not included in
PMAC_HARDWARE_PROB
